src/routes/index.svelte

```diff
@@ -141,9 +209,13 @@
                     min="0"
                     max={maxNumberOfDays - 1}
                     bind:value={currentDay} />
-                <p>Current date: {history[0][0].time[currentDay]}</p>
+                <p>
+                    Current date: {new Date(history[0][0].time[currentDay] * 1000).toISOString()}
```

Comment:
side note: we should send the date array once from the api. not once in each stock object. One for later